frontend/src/pages/distributedWorkloads/global/GlobalDistributedWorkloadsTabs.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
+import { Tabs, Tab, TabTitleText } from '@patternfly/react-core';
+import {
+  DistributedWorkloadsTabId,
+  useDistributedWorkloadsTabs,
+} from './useDistributedWorkloadsTabs';
+
+type GlobalDistributedWorkloadsTabsProps = {
+  activeTabId: DistributedWorkloadsTabId;
+};
+
+const GlobalDistributedWorkloadsTabs: React.FC<GlobalDistributedWorkloadsTabsProps> = ({
+  activeTabId,
+}) => {
+  const navigate = useNavigate();
+  const tabs = useDistributedWorkloadsTabs();
+  const { namespace } = useParams<{ namespace: string }>();
+  return (
+    <Tabs
+      activeKey={activeTabId}
+      onSelect={(_, tabId) => {
+        const tab = tabs.find(({ id }) => id === tabId);
+        if (tab) {
+          const namespaceSuffix = tab.hasProjectParam && !!namespace ? `/${namespace}` : '';
+          navigate(`/distributedWorkloads/${tab.path}${namespaceSuffix}`);
+        }
+      }}
+      aria-label="Workload metrics page tabs"
+      role="region"
+      className="odh-tabs-fix" // TODO mturley hmmmm wasn't there a slack conversation about this recently?
```

Comment:
There was -- Tabs suck for PF layout. Never reference classNames to css files you don't import. Otherwise you might not end up loading the css due to code splitting.

Reply:
Good call. I was copying what's done in PipelineRuns. I'll see if this is needed anymore and either remove it or duplicate the CSS in a file here.